pkg/pillar/types/locationconsts.go

```diff
@@ -25,6 +25,10 @@ const (
 	VolumeClearDirName = ClearDirName + "/volumes"
 	// PersistDebugDir - Location for service specific debug/traces
 	PersistDebugDir = PersistDir + "/agentdebug"
+	// EncryptedCloudInitDirname - Location of encrypted cloud-init user-data
+	EncryptedCloudInitDirname = PersistDir + "/encrypted-cloud-init"
```

Comment:
This doesn't appear to be placing it under vault -- I'm pretty sure it won't be encrypted @eriknordmark